integration_tests/CMakeLists.txt

```
@@ -213,6 +213,7 @@ RUN(NAME vec_01              LABELS cpython llvm)
 RUN(NAME test_str_comparison LABELS cpython llvm)
 
 RUN(NAME generics_01         LABELS cpython llvm)
+RUN(NAME generics_array_01   LABELS llvm)
```

Add a CPython test too:
```suggestion
RUN(NAME generics_array_01   LABELS cpython llvm)
```

---

The problem is that a TypeVar object, in this case T, is not subscriptable in CPython.

---

I see, that is indeed a problem.

I think we should probably design our own "TypeVar" that can handle subscripting, we'll add it to `ltypes` and we have to make sure exactly the same syntax works both in LPython and CPython.